libcodechecker/analyze/store_handler.py

```diff
@@ -243,6 +269,8 @@ def addCheckerRun(session, storage_session, command, name, tag, username,
         return run_id
     except Exception as ex:
         session.close()
+        session2.close()
```

Comment:
what happens to the `lock_session` if there is an exception? what is `session2`?

Reply:
Mistake when I renamed the variable earlier.